app/models/like.rb

```diff
@@ -1,3 +1,9 @@
 class Like < ApplicationRecord
-  belongs_to :post, class_name: 'Post', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'user_id'
+  belongs_to :user, class_name: 'User', foreign_key: 'post_id'
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
```

Comment:
- [ ] **[OPTIONAL]** You can also use the `increment!` method here as demonstrated earlier.